airbyte-integrations/connectors/source-facebook-marketing/integration_tests/test_streams.py

```diff
@@ -60,11 +60,14 @@ def test_streams_with_include_deleted(self, stream_name, deleted_id, config_with
         account_id = config_with_include_deleted["account_id"]
 
         assert states, "incremental read should produce states"
-        for name, state in states[-1].state.data.items():
-            assert "filter_statuses" in state[account_id], f"State for {name} should include `filter_statuses` flag"
+        actual_stream_name = states[-1].state.stream.stream_descriptor.name
+        assert states[-1].state.stream.stream_descriptor == StreamDescriptor(name=stream_name)
+        assert "filter_statuses" in states[-1].state.stream.stream_state.dict()[account_id], f"State for {actual_stream_name} should include `filter_statuses` flag"
 
-        assert deleted_records, f"{stream_name} stream should have deleted records returned"
-        assert is_specific_deleted_pulled, f"{stream_name} stream should have a deleted record with id={deleted_id}"
+        # TODO: This should be converted into a mock server test. There is a 37 month query window and our deleted records
```

Comment:
😡 follow up ticket is https://github.com/airbytehq/airbyte-internal-issues/issues/6552